blog/_posts/2017-09-12-Issue-253.md

```diff
@@ -0,0 +1,134 @@
+
+<br>
+
+#### We Still Want to Do:
+- add HTTP method PATCH to Feedback
+- change field ```weight``` to FloatField
+- add filters to our Feedback + some other limitations
+- refacture
+- writing a Foodsaving-backend beginner guide. We already started [here](https://github.com/mddemarie/Foodsaving-Documentation/blob/master/foodsaving-backend-code-guide.md).
+
+<br>
+
+## Hackathon & Community
+
+We attended HackDays in ImmobillienScount 11.07.-13.07. and we have got very motivated with Stefan: "We can make it!"
+
+Foodsaving founders invited us to their **Hackathon: 14.07.-16.07**. We met here incredibly smart people - Tilmann Becker  (@tiltec), Nick Sellen (@nicksellen) and Janina Abels (@djahnie) who put their time & energy into making the saving a food waste possible.
+
+<br>
+
+![Hackathon in Berlin](/img/blog/2017/hack_1.png) ![Hackathon in Berlin](/img/blog/2017/hack_2.jpeg)   
+_Enjoing saved food and coding together at the Hackathon, Photos by Nick Sellen_
+
+<br>
+
+And we want to thank our great coaches and mentor! Most of them already appeared in our [first blog post](https://github.com/id-gue/summer-of-code/edit/gh-pages/blog/_posts/2017-07-13-print_hello_world.md) – but [Derek Tingle](https://github.com/tingled)  (@tingled) is new and we are very grateful that he spent a lot of time explaining us many related technical topics we needed a help with. You should see how he navigates through the shell – it's amazing!!! :D
+
+[SoundCloud](https://twitter.com/soundcloud) coaching company was incredibly supportive work environment! We would like to thank SoundCloud for providing us a space, working software & hardware, motivational environment and good music! We had very nice time at SoundCloud and SoundCloud people demonstrated their passion for technology that drives them to achieve amazing results. Very enriching experience for us!!!
+
+<br>
+
+![SoundCloud](/img/blog/2017/workspace_1.jpg) ![SoundCloud](/img/blog/2017/workspace_2.JPG) ![SoundCloud](/img/blog/2017/retro_meeting_marie.JPG)
+_SoundCloud is really great place to work! Photos by Ines D. Guett_
+
+<br>
+
+
+## Future
+Our Calls for Papers for [Codemotion Berlin 2017](https://berlin2017.codemotionworld.com/) and [PyCon Germany 2017](https://de.pycon.org/) were successfully selected! Yay! And we are going to have this talk: "A Summer of Foodsharing – Learn How to Code in an Open Source Project" in October 2017. We are going to talk about RailsGirls community, how to get into Open Source and we will present our project Foodsaving.
+
+ 
+@ines will fly to Ghana and Kenya in the autunm and have a look on the tech scene there. Afterwards she would like to become a Product Manager and help to prioritize coding tasks.
```

Comment:
Ines will fly to Ghana and Kenya in the autumn and have a look on the tech scene there. Afterwards, she would like to become a Product Manager and help to prioritize coding tasks.